src/storage/store/stores.rs

```diff
@@ -338,6 +347,63 @@ impl Stores {
         );
         Ok(revoke_events)
     }
+
+    pub fn get_events(
+        &self,
+        start_id: u64,
+        stop_id: Option<u64>,
+        page_options: Option<PageOptions>,
+    ) -> Result<EventsPage, HubError> {
+        HubEvent::get_events(self.db.clone(), start_id, stop_id, page_options)
+    }
+
+    pub async fn prune_events_until(
+        &self,
+        timestamp: u64,
+        throttle: Duration,
+        page_options: Option<PageOptions>,
+    ) -> Result<u32, HubError> {
+        let stop_height = self
+            .shard_store
+            .get_next_height_by_timestamp(timestamp)
+            .unwrap_or_else(|e| {
+                error!(
+                    "Error getting next height by timestamp for shard {}: {}",
+                    self.shard_id, e
+                );
+                None
+            });
+
+        let page_options = page_options.unwrap_or(PageOptions {
+            page_size: Some(PAGE_SIZE_MAX),
+            ..PageOptions::default()
+        });
+
+        if let Some(stop_height) = stop_height {
+            info!(
+                "Pruning events for shard {} older than timestamp: {}, height: {}",
+                self.shard_id, timestamp, stop_height
+            );
+            let start = std::time::Instant::now();
+            let count =
+                HubEvent::prune_events_util(self.db.clone(), stop_height, &page_options, throttle)
+                    .await?;
+            let elapsed = start.elapsed();
+            self.statsd
+                .count_with_shard(self.shard_id, "prune.events", count as u64);
+            self.statsd.time_with_shard(
+                self.shard_id,
```

Comment:
Can we add ms to the metric name? I think it'd be good to do this for all the timing metrics so it's easy to figure out the scale of the y axis.

Reply:
Fixed